src/manifest.c

```diff
@@ -299,6 +299,8 @@ static struct manifest *manifest_from_file(int version, char *component, bool he
 		} else if (c[3] == 'm') {
 			file->is_mix = 1;
 			manifest->is_mix = 1;
+		} else if (c[3] == 'e') {
```

Comment:
is this already defined with mixer-tools team? If so, I think we're good to merge this

Reply:
> is this already defined with mixer-tools team? If so, I think we're good to merge this

@tmarcu @reaganlo can you guys confirm this? ^

Reply:
@castulo Not yet. Will keep you posted when done.

Reply:
It is fine that it is not implemented yet, this change can still be pushed in to swupd-client as long as the mixer team agrees it is the correct approach.

Reply:
This approach has a +1 from me fwiw.

Reply:
@reaganlo it does not need to be implemented yet, I was only asking if that is the way it is going to work. We can merge this without the implementation on the mixer side, it should not affect swupd client at all.

Reply:
@castulo aaa I see. It looks fine. But can you please hold on to merging it until tomorrow. I want to first test and ensure that it does not break anything. Currently working on some high priority Coverity tasks.

Reply:
It will not break anything mixer side. This is just fixing the client to not fail if it encounters the flag.

Reply:
@matthewrsj I was not referring to testing this swupd-client change. What I meant was testing the corresponding implementation in mixer.

Reply:
I don't think we need to wait. Adding flags to manifests is not complicated. It is something we've done before many times.